screen-onboarding/src/main/java/com/ivy/onboarding/viewmodel/OnboardingViewModel.kt

```diff
@@ -274,70 +293,58 @@ class OnboardingViewModel @Inject constructor(
             }.toImmutableList()
     }
 
-    fun onAddAccountsDone() {
-        viewModelScope.launch {
-            TestIdlingResource.increment()
+    private suspend fun onAddAccountsDone() {
+        TestIdlingResource.increment()
 
-            router.accountsNext()
+        router.accountsNext()
 
-            TestIdlingResource.decrement()
-        }
+        TestIdlingResource.decrement()
     }
 
-    fun onAddAccountsSkip() {
-        viewModelScope.launch {
-            TestIdlingResource.increment()
+    private suspend fun onAddAccountsSkip() {
+        TestIdlingResource.increment()
```

Comment:
Delete all `TestIdlingResource` usages in this file. The `TestIdlingResource` is a legacy piece that is no longer needed.